src/Sentry/PlatformAbstractions/NetFxInstallationsEventProcessor.cs

```diff
@@ -23,7 +24,8 @@ internal class NetFxInstallationsEventProcessor : ISentryEventProcessor
         internal static Dictionary<string, string> GetInstallationsDictionary()
         {
             var versionsDictionary = new Dictionary<string, string>();
-            var installations = FrameworkInfo.GetInstallations().ToArray();
+            using var registryKey = Registry.LocalMachine.OpenSubKey(FrameworkInfo.NetFxNdpRegistryKey, false);
+            var installations = FrameworkInfo.GetInstallationsFromRegistryKey(registryKey).ToArray();
```

Comment:
This means we'lll be throwing and catch an exception on each call?
Ideally we'd cache the value of this here once and have the error handling/logging done here instead (remove `catch return null` from the code you've added)

Reply:
Can probably just wrap the whole thing in a `Lazy<T>`

Reply:
Changed the flow so it'll only throw only once and log the exception when an Event is processed by NetFxInstallationsEventProcessor